CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## NEXT
```

Comment:
Also need to add some doc about set labelFn

Reply:
done